src/data/fintype/basic.lean

```diff
@@ -1299,8 +1299,8 @@ by { ext, simp [finset.mem_powerset_len] }
   fintype.card {s : finset α // s.card = k} = nat.choose (fintype.card α) k :=
 by simp [fintype.subtype_card, finset.card_univ]
 
-@[simp] lemma set.to_finset_univ [fintype α] :
-  (set.univ : set α).to_finset = finset.univ :=
+@[simp] lemma set.to_finset_univ [hu : fintype (set.univ : set α)] [fintype α] :
+  @set.to_finset _ (set.univ : set α) hu = finset.univ :=
```

Comment:
Are the `@`s actually needed here?

Reply:
@eric-wieser Do you think the intent is clearer without it?

Reply:
Yes I think it would be clearer, because if it didn't actually use this argument the linter would be complaining anyway.

If you want to be clear, a docstring explaining that this argument is here to make the lemma more general than it would be from just inferring it from the other fintype argument.

Reply:
It's actually necessary -- typeclass inference gives the `subtype.fintype` instance for `set.to_finset` otherwise.

(In addition to that, there's an elaboration bug. I'll mention this on Zulip.)